### PR TITLE
fix: refine custom copilot qm wordings

### PR DESCRIPTION
**Pull request by kuojianlu**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/26356837

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -293,7 +293,7 @@
   "core.createProjectQuestion.projectType.copilotPlugin.label": "Copilot Plugin",
   "core.createProjectQuestion.projectType.copilotPlugin.title": "Copilot Plugin",
   "core.createProjectQuestion.projectType.copilotPlugin.placeholder": "Select an option",
-  "core.createProjectQuestion.projectType.customCopilot.detail": "Build intelligent chatbot in Microsoft Teams easily using Teams AI Library",
+  "core.createProjectQuestion.projectType.customCopilot.detail": "Build intelligent chatbot as your own copilot in Microsoft Teams easily using Teams AI Library",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:296]: Any specific reason to mention 'as your own copilot'?

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -312,8 +312,8 @@
   "core.createProjectQuestion.capability.customCopilotBasicOption.detail": "Build a basic AI chatbot in Teams",
   "core.createProjectQuestion.capability.customCopilotRagOption.label": "Chat With Your Data",
   "core.createProjectQuestion.capability.customCopilotRagOption.detail": "Expand AI bot's knowledge with your content to get accurate answers to your questions",
-  "core.createProjectQuestion.capability.customCopilotAssistantOption.label": "AI Assistant",
-  "core.createProjectQuestion.capability.customCopilotAssistantOption.detail": "Build intelligent chatbot in Teams to complete everyday tasks efficiently",
+  "core.createProjectQuestion.capability.customCopilotAssistantOption.label": "AI Agent",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:315]: Going forward, are we calling 'AI assistant' as 'AI agent' everywhere?

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (kuojianlu) [reply packages/fx-core/resource/package.nls.json:296]: To be consistent with the title "Custom Copilot", i.e., let users know the bot is built as their own copilot in Teams.

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -312,8 +312,8 @@
...
+  "core.createProjectQuestion.capability.customCopilotAssistantOption.detail": "Build an AI agent in Teams that can reason over LLMs to take a sequence of actions",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:316]: This is complex and will add cognitive load to users.
Proposed: Build an AI agent in Teams that can make decisions and perform actions based on LLM reasoning.

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (supkasar) [reply packages/fx-core/resource/package.nls.json:296]: Got it.
To improve readability, remove the word 'easily'.

Comment (kuojianlu) [reply packages/fx-core/resource/package.nls.json]: Updated.

File: packages/fx-core/resource/package.nls.json (hunk 3 above)
Comment (kuojianlu) [reply packages/fx-core/resource/package.nls.json]: Agree with you. Thanks, updated.

**Review by supkasar — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**